usecases/mail-sender/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+mail.from = noreply@notification.us-ashburn-1.oci.oraclecloud.com
```

Comment:
Maybe an environment variable could be used for the OCI region (or commit this as using a placeholder value that requires the user to update - even if it fails as-is, requiring them to set it to the correct value).

Reply:
These are now replaced with environment variables.